core/repository/sparql/src/main/java/org/eclipse/rdf4j/repository/sparql/SPARQLConnection.java

```diff
@@ -972,4 +996,8 @@ protected Statement convert(BindingSet b) throws QueryEvaluationException {
 		};
 	}
 
+	private ModelFactory getModelFactory() {
+		return modelFactory;
+	}
```

Comment:
Do we need a private getter?

Reply:
Not necesarily, I think I originally made it public and then changed my mind :)